.gitignore

```diff
@@ -0,0 +1 @@
+octomap/octomap.cpp
```

Comment:
seems cython generates this file `octomap/octomap.cpp` so i assime we want to ignore?